drivers/Aqara/aqara-presence-sensor/src/init.lua

```diff
@@ -64,7 +59,9 @@ local function create_sse(driver, device, credential)
     end
 
     eventsource.onopen = function()
+      log.info_with({ hub_logs = true }, string.format("Eventsource open: dni= %s", device.device_network_id))
       device:online()
+      status_update(driver, device)
```

Comment:
Should the error case be checked and logged or handled here?

Reply:
Thank you for letting me know. We added error handling logic for this.
I called status_update again in case it failed due to a simple timing issue.